src/rpc/blockchain.cpp

```diff
@@ -45,8 +45,8 @@ void ScriptPubKeyToJSON(const CScript& scriptPubKey, UniValue& out, bool fInclud
 
 double GetDifficulty(const CBlockIndex* blockindex)
 {
-    // Floating point number that is a multiple of the minimum difficulty,
-    // minimum difficulty = 1.0.
+    // Floating point number that is a multiple of the main net minimum
```

Comment:
Would be even better to turn it into a doxygen comment and move it outside the function. E.g.
```
/** Returns a floating point number that is a multiple of the main net minimum
 * difficulty (4295032833 hashes).
 */
```